luxonis_train/loaders/base_loader.py

```diff
@@ -154,7 +154,11 @@ def collate_fn(
         if label_type in [LabelType.CLASSIFICATION, LabelType.SEGMENTATION]:
             out_labels[task] = torch.stack(annos, 0), label_type
 
-        elif label_type in [LabelType.KEYPOINTS, LabelType.BOUNDINGBOX]:
+        elif label_type in [
+            LabelType.KEYPOINTS,
+            LabelType.BOUNDINGBOX,
+            LabelType.OBOUNDINGBOX,
```

Comment:
`LabelType.ORIENTED_BOUNDINGBOX` would be a better name so it doesn't look like a typo (though that's a comment for the `luxonis-ml` side of the implementation)